android/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
+
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.java]
+charset = utf-8
+indent_style = tab
```

Comment:
Man, what did I copy & paste there... replace this nonsense with https://github.com/saltyrtc/saltyrtc-task-webrtc-java/blob/new-api/.editorconfig